### DISCUSSION TITLE
MTV | (Colombia)

### DISCUSSION THREAD

**Discussion by R4IN2007** [iptv-org/community#353] [Show and tell]
### Channel Name

MTV

### Channel ID

MTVNorth.us

### Stream URL

http://190.61.41.182:8000/play/a1iz/index.m3u8 (New m3u8 Link)

### Please confirm the following

- [X] I have read [Contributing Guide](https://github.com/iptv-org/iptv/blob/master/CONTRIBUTING.md#request-a-channel)

**Comment by spotanjo3** [comment]
This is Spanish. What about USA version ?

**Comment by BellezaEmporium** [comment]
IP doesn't answer.

**Comment by freearhey** [comment]
http://190.61.41.182:8000/play/a1iz/index.m3u8 (offline)
<img width="334" alt="image" src="https://user-images.githubusercontent.com/7253922/232229414-fc2a4310-4178-401b-9721-c23028d5d1d1.png">
